e2e/libs/persistent_volume_claim/crd.py

```diff
@@ -0,0 +1,40 @@
+from kubernetes import client
+
+from persistent_volume_claim.base import Base
+
+from utility.utility import get_retry_count_and_interval
+from utility.utility import logging
+
+
+class CRD(Base):
+
+    def __init__(self):
+        self.core_v1_api = client.CoreV1Api()
+        self.retry_count, self.retry_interval = get_retry_count_and_interval()
+
+    def get(self, claim_name, claim_namespace="default"):
+        return self.core_v1_api.read_namespaced_persistent_volume_claim(
+            name=claim_name,
+            namespace=claim_namespace,
+        )
+
+    def expand(self, claim_name, size, namespace="default"):
```

Comment:
Besides `expand`, will we support any other operation on pvc needed to be added into this class?

Reply:
This is all we need for the scenarios in this PR. Additional operations will be determined as more cases are added. These can be addressed later.

Reply:
I think it's more organized to have a class for each CR. If it's unnecessary after we add other cases, we can reorganize it later.